libs/community/langchain_community/llms/huggingface_text_gen_inference.py

```diff
@@ -129,8 +131,17 @@ def build_extra(cls, values: Dict[str, Any]) -> Dict[str, Any]:
 
     @root_validator()
     def validate_environment(cls, values: Dict) -> Dict:
-        """Validate that python package exists in environment."""
-
+        """Validate that api key and python package exists in environment."""
+        huggingfacehub_api_token = get_from_dict_or_env(
+            values, "huggingfacehub_api_token", "HUGGINGFACEHUB_API_TOKEN"
+        )
```

Comment:
will raise an error if no default is provided and token isn't provided, can we do
```suggestion
        huggingfacehub_api_token = get_from_dict_or_env(
            values, "huggingfacehub_api_token", "HUGGINGFACEHUB_API_TOKEN", default=""
        )
```